spec/index.bs

```diff
@@ -410,6 +410,15 @@ The [=subresources=] of textures included in the views provided to
 are considered to have {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
 for the [=usage scope=] of this render pass.
 
+The <dfn dfn>physical size</dfn> of a {{GPUTexture}} [=subresource=] is the dimension of the {{GPUTexture}}
+[=subresource=] in pixels that includes the possible extra paddings to form complete [=texel blocks=] in the
+[=subresource=]. The extra paddings only exists in a compressed texture mipmap whose {{GPUTexture/[[textureSize]]}}
+at [=mipmap level=] 0 are multiple of [=texel block size=], but its subdivided levels are not. For example,
+considering a texture in BC format whose {{GPUTexture/[[textureSize]]}} is {60, 60, 1}, when sampling the texture
+at [=mipmap level=] 2, the sampling hardware uses {15, 15, 1} as the size of the [=subresource=], while its
+[=physical size=] is {16, 16, 1} as the block-compression algorithm can only operate on 4x4 [=texel blocks=]. There
+is no extra padding for the {{GPUTexture}} [=subresources=] with non-compressed texture formats.
```

Comment:
I think it would be more clear to make the difference between block-based texture formats and pixel-based formats because eventually we might have other formats like YUV 420 that are block-based but not traditionally thought of as compressed.

Reply:
Done